### PR TITLE
updated tree for front controller

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: create_framework/front-controller.rst [hunk 1]
```diff
@@ -146,11 +146,13 @@ web root directory:
 
     example.com
     ├── composer.json
+    ├── composer.lock    
     │   src
```
Comment (javiereguiluz) [inline create_framework/front-controller.rst]: Here again this line should be:

```
├── src
```

Instead of:

```
│   src
```

Comment (OskarStark) [inline create_framework/front-controller.rst]: :+1:

**Comment by OskarStark** [comment]
can we merge this @WouterJ @xabbuh

**Comment by WouterJ** [comment]
Yes (will do it another time though, just stopped merging)

**Comment by OskarStark** [comment]
(y) thank you @WouterJ

**Comment by xabbuh** [comment]
Thank you Oskar.

**Comment by OskarStark** [comment]
:+1:

**Pull request closed without merging**